library/tasks/ModifyGtf.wdl

```diff
@@ -0,0 +1,51 @@
+task ReplaceGeneNameWithGeneID {
+  File original_gtf
+
+  #runtime values
+  String docker = "quay.io/humancellatlas/modify-gtf:0.1.0"
+  Int machine_mem_mb = 8250
+  Int cpu = 1
+  Int disk = ceil(size(original_gtf, "Gi") * 2) + 1
```

Comment:
What unit is `Gi`? I've only seen `GB` and `GiB`

Reply:
All of Optimus actually uses the `Gi` syntax, so I would rather leave it consistent to be fixed in a different PR. I share the same confusion.

Reply:
That's fair.